node/clients/PvtCheckout.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  InstanceOptions,
+  IOContext,
+  JanusClient,
+  RequestConfig,
+} from '@vtex/api'
+
+import { checkoutCookieFormat, statusToError } from '../utils'
+
+export class PvtCheckout extends JanusClient {
+  constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(ctx, {
+      ...options,
+      headers: {
+        ...options?.headers,
+        ...(ctx.authToken ? { VtexIdclientAutCookie: ctx.authToken } : null),
+        'x-vtex-user-agent': ctx.userAgent,
+      },
+    })
+  }
+
+  private getCommonHeaders = () => {
+    const { orderFormId, segmentToken, sessionToken } = this
+      .context as CustomIOContext
+
+    const checkoutCookie = orderFormId ? checkoutCookieFormat(orderFormId) : ''
```

Comment:
can we _not_ use the orderFormId on this client? because we already are using `authToken` and if someones mistakenly uses this client to update something on the orderForm it will cause the same issue as the last one :sadge: